src/index.html

```diff
@@ -9,6 +12,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
```

Comment:
it's better to use `header` name for better search
```suggestion
    <header class="header">
```